### PR TITLE
Ignore pull request on repos owned by same user

### PR DESCRIPTION
**Pull request by printminion**
Fixes issue of users creating pull requests on own repos

Changes: Compare username with user name in pull request URL (case insensitive)

### CODE REVIEW

**Comment by lex111** [comment]
@printminion this is a good suggestion, it does **not** contradict the rules, and such PR will be taken into account. From Hacktoberfest support:

> You can contribute to your own repos. But, we encourage you to use Hacktoberfest as an opportunity to find a new project to contribute to.

But I think it is better able to visually highlight such PR's? Or to show separately from others?

**Comment by jenkoian** [comment]
Agree with @lex111 some visual marker to show them as slightly less valued or something would be better than ignoring them completely.

File: controllers/index.js [hunk 1]
```diff
@@ -30,6 +30,14 @@ exports.index = (req, res) => {
 
                 const hacktoberFestLabels = _.filter(event.labels, label => label.name.toLowerCase() === 'hacktoberfest');
 
+
+                //ignore pull requests on own repos
+                const repoOwner = repo.replace('https://github.com/', '').split('/')[0];
+
+                if (username.toUpperCase() === repoOwner.toUpperCase()) {
```
Comment (fobos) [inline controllers/index.js]: Every cycle in `_.each` you call `username.toUpperCase`. It's better to call it only once before `_.each`.

**Comment by jenkoian** [comment]
Closing in prep for 2018 rewrite.

**Pull request closed without merging**